### ISSUE TITLE
Change JSON Structure

### ISSUE THREAD

**Issue by 13mralex** [pirxpilot/liftie#43]
My reverse engineering skills aren't the greatest...I am trying to index the lift names and status, but since they aren't numbered, I’m having trouble. Is there a way to change it like described below?

**Current:**
<img width="440" alt="Screen Shot 2021-12-24 at 10 08 57 PM" src="https://user-images.githubusercontent.com/32402821/147376546-f419db12-8caf-47b5-83e6-19e42a4da9ba.png">
**Suggested:**
<img width="443" alt="Screen Shot 2021-12-24 at 10 10 23 PM" src="https://user-images.githubusercontent.com/32402821/147376558-a9cf6029-95cd-4820-95aa-b538ec4c2ac7.png">

Basically, just have each lift status have its own unique, numbered index rather than the name of the lift be the index.

**Comment by pirxpilot** [comment]
Changing status format for Liftie server would would break Liftie API. It would mean rewriting all the tests, reworking the web client and it would require changes for all current Liftie API users. Not to mention it would also make status payloads bigger.

I am also not sure what you mean by _index the lift names and status_

Could you do that with something like:

    Object.entries(lifts.status).map(([name, status]) => ({name, status}))

**Comment by 13mralex** [comment]
I should've mentioned I am running a local copy. Which file do I need to edit to achieve this?
